jaeger-context/src/main/java/com/uber/jaeger/context/TracingUtils.java

```diff
@@ -14,26 +14,37 @@
 
 package com.uber.jaeger.context;
 
-import io.opentracing.util.GlobalTracer;
-
+import io.opentracing.NoopTracer;
+import io.opentracing.NoopTracerFactory;
 import java.util.concurrent.ExecutorService;
 
+/**
+ * TracingUtils is going to be deprecated. To use its method please provide a tracer via
+ * {@link com.uber.jaeger.context.TracingUtils#setTracer(io.opentracing.Tracer)}.
+ */
+@Deprecated
 public class TracingUtils {
-  private static TraceContext traceContext = new ActiveSpanSourceTraceContext(GlobalTracer.get());
+  private static io.opentracing.Tracer tracer = NoopTracerFactory.create();
```

Comment:
I think `null` would do just as well here, even safer in case one of the business methods forgets to call assertTracerRegistered().

Reply:
+1